src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -134,8 +132,10 @@ class MediaController(override val config: TableauxConfig,
       _ <- fileModel.deleteById(uuid)
       _ <- {
         import FutureUtils._
+
         promisify({ p: Promise[Unit] =>
-          vertx.fileSystem.delete(path.toString(), {result => p.success()})
+          // succeed even if file doesn't exist
+          vertx.fileSystem.delete(path.toString(), { result => p.success(()) })
```

Comment:
Why wouldn't we have the permission? We create these files and we create the `uploads/`. As long as we have `w`-permission on the `uploads/` we can delete uploaded files. And if the file is already missing we are fine. If someone changes the permission on `uploads/` we can't do anything about it. I will log an error in this case.
One problem remains, if we don't have `w`-permission on `uploads/` no error is replied on file-upload requests.